packages/data-transport-layer/src/services/server/service.ts

```diff
@@ -106,29 +106,46 @@ export class L1TransportServer extends BaseService<L1TransportServerOptions> {
   private _initializeApp() {
     // TODO: Maybe pass this in as a parameter instead of creating it here?
     this.state.app = express()
+    if (this.options.ethNetworkName) this._initMonitoring()
+    this.state.app.use(cors())
+    this._registerAllRoutes()
+    // Error handling needs to be initialized last
+    if (this.options.ethNetworkName)
```

Comment:
Minor nit on using curly braces here and above. I think it's useful to be able to delineate where an `if` statement applies and makes it easier to add new things to the interior of the statement if necessary. Otherwise LGTM.